packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -264,4 +265,59 @@ describe('<XGrid />', () => {
       expect(getColumnValues()).to.deep.equal(['Puma', 'Nike', 'Adidas']);
     });
   });
+  describe('state', () => {
+    it('should allow to control the state using useState', () => {
```

Comment:
If the feature is available in the MIT version, I think that we should move the test to the DataGrd.test.jsx file.